index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+  <!-- title  -->
+  <title>Talent Land Mexico 2023</title>
+  <link rel="icon" type="image/x-icon" href="./assets/favicon.png">
+  <!-- CSS Style -->
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- CDN Font -->
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">                
+  <!-- Google Font -->
+  <script src="https://kit.fontawesome.com/e52d74ec85.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Rubik+Dirt&display=swap" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div>
+      <ul>
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div>
+      <img src="./assets/logo-talent-land.png" alt="">
+      <span id="openOnClick"><a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776</a></span>
+      <nav>
+        <ul>
+          <li><a href="./about.html"> About</a></li>
```

Comment:
- [optional] you can make your menu full width and height so it serves its purpose when it comes up which is for a user to use it to navigate to different page sections. as it is it helps just moving from about to home page of the mobile version. a user should not be able to move around the page unless the menu is closed.
![Screenshot (992)](https://user-images.githubusercontent.com/65298048/233720152-0608f2d4-3197-4fc6-8bab-5dcc4af967ee.png)